### PR TITLE
Add support for ENABLE and DISABLE on ALTER TABLE for pg

### PR DESCRIPTION
**Pull request by tobyhede**
Add support for the following pg syntax

```
# enable
ALTER TABLE tab ENABLE ALWAYS RULE rule_name
ALTER TABLE tab ENABLE ALWAYS TRIGGER trigger_name
ALTER TABLE tab ENABLE REPLICA TRIGGER trigger_name
ALTER TABLE tab ENABLE REPLICA RULE rule_name
ALTER TABLE tab ENABLE ROW LEVEL SECURITY
ALTER TABLE tab ENABLE RULE rule_name
ALTER TABLE tab ENABLE TRIGGER ALL
ALTER TABLE tab ENABLE TRIGGER USER
ALTER TABLE tab ENABLE TRIGGER trigger_name

# disable
ALTER TABLE tab DISABLE ROW LEVEL SECURITY
ALTER TABLE tab DISABLE RULE rule_name
ALTER TABLE tab DISABLE TRIGGER ALL
ALTER TABLE tab DISABLE TRIGGER USER
ALTER TABLE tab DISABLE TRIGGER trigger_name
```

### CODE REVIEW

File: src/tokenizer.rs [hunk 1]
```diff
@@ -727,10 +727,7 @@ impl<'a> Tokenizer<'a> {
                     // match binary literal that starts with 0x
                     if s == "0" && chars.peek() == Some(&'x') {
                         chars.next();
-                        let s2 = peeking_take_while(
-                            chars,
-                            |ch| matches!(ch, '0'..='9' | 'A'..='F' | 'a'..='f'),
-                        );
+                        let s2 = peeking_take_while(chars, |ch| ch.is_ascii_hexdigit());
```
Comment (tobyhede) [inline src/tokenizer.rs:730]: New clippy rule dropped.

File: src/tokenizer.rs [hunk 2]
```diff
@@ -1077,7 +1074,7 @@ impl<'a> Tokenizer<'a> {
                 match chars.peek() {
                     Some('$') => {
                         chars.next();
-                        for (_, c) in value.chars().enumerate() {
+                        for c in value.chars() {
```
Comment (tobyhede) [inline src/tokenizer.rs:1077]: New clippy rule dropped.

File: src/dialect/mod.rs [hunk 3]
```diff
@@ -349,6 +349,7 @@ mod tests {
             }
         }
 
+        #[allow(clippy::needless_raw_string_hashes)]
```
Comment (alamb) [inline src/dialect/mod.rs:352]: maybe we can fix this in a follow on PR (as in remove the clippy allow and then fix the tests to conform to the new clippy opinions)

**Review by alamb — APPROVED** [review]
Looks good to me -- thank you for the contribution @tobyhede

File: src/dialect/mod.rs (hunk 3 above)
Comment (tobyhede) [reply src/dialect/mod.rs:352]: Yeah, is a good idea. I tried to fix, but without the raw the parser errors with `Unterminated string literal`. 
I wasn't sure enough of the underlying intent to make more a dramatic change.

**Pull request merged**